src/remoteconfig/versionslist.ts

```diff
@@ -1,8 +1,14 @@
-import api = require("../api");
+import { remoteConfigApiOrigin } from "../api";
+import { Client } from "../apiv2";
 import { FirebaseError } from "../error";
 import { ListVersionsResult } from "./interfaces";
 import { logger } from "../logger";
 
+const apiClient = new Client({
+  urlPrefix: remoteConfigApiOrigin,
+  apiVersion: "v1",
+});
```

Comment:
Worth putting this in `remoteconfig/client.ts` like Hosting has?

Reply:
I go back and forth - I only thought of making a single client recently. Might be a good thing to do, but I'll leave it be unless someone from RC says otherwise.

Reply:
I think this would be a nice to have 👍 would def be a bit more cleaner